### PR TITLE
feat/move-balance-into-accounts-slice

### PR DESCRIPTION
**Pull request by mateuszjasiuk**
Few things happen here :)
- balance is moved to accounts
- we add chainIds to the accounts initial state
- router is updated
- we query accounts and balances in App if we are connected to extension

### CODE REVIEW

File: apps/namada-interface/src/App/AccountOverview/DerivedAccounts/DerivedAccounts.tsx [hunk 1]
```diff
@@ -231,13 +176,11 @@ const DerivedAccounts = ({ setTotal }: Props): JSX.Element => {
...
                 <TokenBalances>
-                  {tokens.map((tokenBalance) => {
-                    const { balance, token } = tokenBalance;
-
+                  {Object.entries(balance).map(([token, amount]) => {
                     return (
                       <TokenBalance key={`${address}-${token}`}>
                         <TokenIcon
-                          src={getAssetIconByTheme(token)}
+                          src={getAssetIconByTheme(token as TokenType)}
```
Comment (mateuszjasiuk) [inline apps/namada-interface/src/App/AccountOverview/DerivedAccounts/DerivedAccounts.tsx:183]: TODO take a look at

File: apps/namada-interface/src/App/Staking/NewBondingPosition/NewBondingPosition.tsx [hunk 2]
```diff
@@ -54,34 +53,40 @@ type Props = {
...
+  const currentNAMBalance = currentBalance["NAM"] || 0;
 
   const handleAddressChange = (
     e: React.ChangeEvent<HTMLSelectElement>
   ): void => {
     const address = e.target.value;
-    setAddress(address);
-    setCurrentBalance(balance[address]);
+    const addrWithBalance = addressesWithBalance.find(
+      ({ address: a }) => address === a
```
Comment (mateuszjasiuk) [inline apps/namada-interface/src/App/Staking/NewBondingPosition/NewBondingPosition.tsx:84]: Fix it

File: apps/namada-interface/src/App/Token/IBCTransfer/IBCTransfer.tsx [hunk 3]
```diff
@@ -80,60 +75,34 @@ const IBCTransfer = (): JSX.Element => {
...
+        } (${amount} ${tokenType})`,
+      }));
+    }
+  );
 
-  const { tokenType } = tokenBalances[0] || {};
-  const balance =
-    tokenBalances.find((balance) => balance.address === selectedAccountAddress)
-      ?.balance || 0;
+  const [token, setToken] = useState<TokenType>("NAM");
```
Comment (mateuszjasiuk) [inline apps/namada-interface/src/App/Token/IBCTransfer/IBCTransfer.tsx:100]: currency.symbol token

**Review by jurevans — APPROVED** [review]
Per our discussion, this looks good!

**Pull request merged**